bench/siphash.cc

```
@@ -0,0 +1,46 @@
+#include <openssl/base.h>
+#include <openssl/err.h>
+#include <openssl/evp.h>
+#include <openssl/siphash.h>
+
+#include "./internal.h"
+
+namespace {
+
+void BM_SpeedSipHash(benchmark::State &state) {
+  uint64_t key[2] = {0};
+  size_t input_size = state.range(0);
```

Clang (14.0.6)

[warning] variable 'input_size' is not initialized

(cppcoreguidelines-init-variables)